flatten-array.md

```diff
@@ -1,8 +1,11 @@
-The challenge is to write a function that accepts an arbitrarily-deep nested list-like structure and returns a flattened structure without any null/nill values. 
- 
-For Example
+The challenge is to write a function that accepts an arbitrarily-deep nested list-like structure and returns a flattened structure without any null/nill values.
+
+For example:
 
 input: [1,[2,3,null,4],[null],5]
 
 output: [1,2,3,4,5]
 
+## Note
+
+Implement the function yourself. Do not use something else to perform the flattening for you.
```

Comment:
@kytrinyx Ok, I didn't know that for some tracks it might be more useful. Of course, if we can't decide whether to add such note or not, it is better to let people choose the approach they want ;)